controllers/storageclassclaim_controller.go

```diff
@@ -358,6 +358,9 @@ func (r *StorageClassClaimReconciler) reconcilePhases() (reconcile.Result, error
 				if err != nil {
 					return reconcile.Result{}, fmt.Errorf("failed to create or update StorageClass: %s", err)
 				}
+			case "CephBlockPoolRadosNamespace":
+				csiClusterConfigEntry.CephRBD = new(csi.CephRBDSpec)
+				csiClusterConfigEntry.CephRBD.RadosNamespace = resource.Name
```

Comment:
@jarrpa Is this old code? If I remember correctly, we agreed that the resource array indicates actual resources to create or edit. We are not creating or editing `CephBlockPoolRadosNamespace` on the client so we should do all to processing to aggregate into client side resources on the provider side